File: src/main/java/com/ekart/model/AddressInfo.java
package com.ekart.model;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class AddressInfo {

  public String street;
  public int apartment;
  public String city;
  public String postalCode;
  public String phoneNumber;
}

File: src/main/.net/Model/AddressInfo.cs
namespace ShoppingApp
{
    public sealed class AddressInfo
    {
        public string Street { get; }
        public int Apartment { get; }
        public string City { get; }
        public string PostalCode { get; }
        public string PhoneNumber { get; }

        public AddressInfo(string street, int apartment, string city, string postalCode, string phoneNumber)
        {
            Street = street;
            Apartment = apartment;
            City = city;
            PostalCode = postalCode;
            PhoneNumber = phoneNumber;
        }

        // TODO: Add operations
        //
        // void ChangeAddress(...) { }
        //
        // void ChangePhoneNumber(...) { }
    }
}